internal/pkg/pct/install.go

```diff
@@ -12,6 +17,14 @@ import (
 	"gopkg.in/yaml.v2"
 )
 
+type HTTPClient interface {
```

Comment:
nit:
```suggestion
type HTTPClientI interface {
```

...to keep in line with our naming convention